snowcrash.gyp

```diff
@@ -10,6 +10,11 @@
         'ext/markdown-parser/ext/sundown/src',
         'ext/markdown-parser/ext/sundown/html'
       ],
+      'direct_dependent_settings' : {
+          'include_dirs': [
+            'ext/markdown-parser/ext/sundown/src',
+          ],
```

Comment:
No, I need no. `html` is required just for build `libsundown`. It is not exposed into public interface and need no to be exposed in inherited `include_dirs`